.github/CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # Contributing Guidelines
 
--   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is _#open-source-progs_.
```

Comment:
#open-source-progs - missing link to Open Source Programs Zulip stream https://anitab-org.zulipchat.com/#streams/237907/open-source-progs  (as in markdown - [ name ]( link )   ), no need of italics

Pls add the link, thank you